docs/features/autoupdater.rst

```diff
@@ -30,6 +30,27 @@ in ``site.mk``, care must be taken to pass the same ``GLUON_RELEASE`` to ``make
 as otherwise the generated manifest will be incomplete.
 
 
+Manifest format
+------------------------
+
+The manifest starts with a short header, followed by the list of firmwares and signatures.
+The header contains the following information:
+
+.. code-block:: sh
+
+    BRANCH=stable
+    DATE=2020-10-07 00:00:00+02:00
+    PRIORITY=7
+
+- ``BRANCH`` is the autoupdater branch name that needs to match the nodes configuration.
+- ``DATE`` specifies when the update time period for the update begins. Nodes might not always have a working NTP
+  configuration, which is why a fallback mechanism exists, that automatically updates if ``DATE`` is at least 24h
+  in the past.
```

Comment:
i'm not sure if this is completely correct or might be misleading, as it doesn't mention that regular updates only happen between 04:00 and 04:59 , see code comment here:
https://github.com/freifunk-gluon/gluon/blob/master/package/gluon-autoupdater/luasrc/lib/gluon/upgrade/500-autoupdater#L54-L57

Reply:
Please go ahead and make a suggestion.